core/unit_test/TestTeamTeamSize.hpp

```diff
@@ -172,4 +172,51 @@ TEST(TEST_CATEGORY, team_policy_max_recommended) {
       max_scratch_size / 2);
 }
 
+template <typename TeamHandleType, typename ReducerValueType>
+struct PrintFunctor1 {
+  KOKKOS_INLINE_FUNCTION void operator()(const TeamHandleType& team,
+                                         ReducerValueType&) const {
+    printf("Test %i %i\n", (int)team.league_rank(), (int)team.team_rank());
```

Comment:
Any reason for C-style casting here? E.g., `int(team.league_rank())` should work fine and not raise the red flags that C-style casts should raise with the reader.

Reply:
> Any reason for C-style casting here?

Consistency with the rest of the code in the file.  The same goes with the `typedef`s.

Reply:
Anyway we are not going to argue about this.  @dhollman please review the latest changes used by Daniel.